test/document_stores/test_memory.py

```diff
@@ -73,6 +74,19 @@ def test_update_bm25(self, documents):
         assert isinstance(bm25_representation, BM25)
         assert bm25_representation.corpus_size == ds.get_document_count()
 
+    @pytest.mark.integration
+    def test_update_bm25_table(self):
+        table_doc = Document(
+            content=pd.DataFrame(columns=["id", "text"], data=[["1", "This is a test"], ["2", "This is another test"]]),
+            content_type="table",
+        )
+        ds = InMemoryDocumentStore(use_bm25=False)
+        ds.write_documents([table_doc])
+        ds.update_bm25()
```

Comment:
I believe we shouldn't need this call, the BM25 index is updated automatically in the `write_documents` call. If it's required, then I'd consider it a bug.